### ISSUE TITLE
Design | Direct Deposit | Update  bank fraud alert for better accessibility

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#63493]
## Background 

The note/alert should be moved to the top of the screen and should be it's own H2 since it is relevant to all sections of the direct deposit screen. Screen reader users who navigate by headings are liable to miss the note.

For more context, the current location of the note makes it appear to be only for the first `h3`, when I believe it is meant to apply for all the sections instead.

_Originally posted by @joshkimux in https://github.com/department-of-veterans-affairs/va.gov-team/issues/19495#issuecomment-775532352_
<img width="1020" alt="Screen Shot 2021-02-08 at 6 25 31 PM" src="https://user-images.githubusercontent.com/14154792/107294308-2e758300-6a3b-11eb-89e9-9d327880754f.png">

## Tasks 
- [ ] 
- [ ] Update the mocks 

## Acceptance Criteria 
- [ ] ready for FE

**Comment by andaleliz** [comment]
@mtcA6 this has been reviewed by content and accessibility, and is ready for FE.

Both mock-ups are on [this page](https://www.sketch.com/s/1a920e73-1dcb-47c4-aae8-08656756c131/p/BBCC1A95-A8A1-410B-805E-C2F8B347C241/canvas). Once they've been validated on the FE, let's make a ticket to update all the artboards in the mobile and desktop pages to reflect the latest. 

We don't really have a great process for making sure work in progress and prod are represented in Sketch. Since our use case docs link to the Sketch files, I'd rather keep them aligned w/ prod as much as we can.  I kinda messed this up with the select/radio buttons but I don't think it's worth going back to un-do, and then re-do all of those.
